### PR TITLE
Move some token filter migration notes

### PR DESCRIPTION
**Pull request by cbuescher**
The removal of the "nGram" and "edgeNGram" token filter was filed in a new
migrate_8_0/mappings.asciidoc document but it should better be in an
analysis.asciidoc section and file.

### CODE REVIEW

**Comment by cbuescher** [comment]
Just a small correction to #38911.

File: docs/reference/migration/migrate_8_0.asciidoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ See also <<release-highlights>> and <<es-release-notes>>.
 
 coming[8.0.0]
 
-* <<breaking_80_mappings_changes>>
```
Comment (jimczi) [inline docs/reference/migration/migrate_8_0.asciidoc]: We have entries in this file now so it should be preserved.

File: docs/reference/migration/migrate_8_0.asciidoc [hunk 2]
```diff
@@ -29,4 +29,4 @@ Elasticsearch 7.x in order to be readable by Elasticsearch 8.x.
 
 =========================================
 
-include::migrate_8_0/mappings.asciidoc[]
```
Comment (jimczi) [inline docs/reference/migration/migrate_8_0.asciidoc]: Same here

**Review by jimczi — CHANGES_REQUESTED** [review]

**Comment by cbuescher** [comment]
@jimczi I added the changes you asked for

**Review by javanna — APPROVED** [review]

**Pull request merged**